packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -50,6 +51,12 @@ export interface PickersShortcutsProps<TValue> extends ExportedPickersShortcutPr
   isValid: (value: TValue) => boolean;
 }
 
+const PickersShortcutsRoot = styled(List, {
+  name: 'MuiPickersLayout',
+  slot: 'Shortcuts',
+  overridesResolver: (_, styles) => [styles.shortcuts],
```

Comment:
Complete nitpick: wrapping to an array is redundant. 🤔 
Same for `PickersActionBar`. 😉 
```suggestion
  overridesResolver: (_, styles) => styles.shortcuts,
```